bitnami/influxdb/README.md

```diff
@@ -51,169 +51,208 @@ The command removes all the Kubernetes components associated with the chart and
+| `relay.service.type`                     | Kubernetes service type (`ClusterIP`, `NodePort` or `LoadBalancer`)                                                          | `ClusterIP`                                             |
+| `relay.service.port`                     | InfluxDB Relay<sup>TM</sup> HTTP port                                                                                        | `9096`                                                  |
+| `relay.service.nodePort`                 | Kubernetes HTTP node port                                                                                                    | `""`                                                    |
+| `relay.service.annotations`              | Annotations for InfluxDB Relay<sup>TM</sup> service                                                                          | `{}`                                                    |
+| `relay.service.loadBalancerIP`           | loadBalancerIP if service type is `LoadBalancer`                                                                             | `nil`                                                   |
+| `relay.service.loadBalancerSourceRanges` | Address that are allowed when service is LoadBalancer                                                                        | `[]`                                                    |
+| `relay.service.clusterIP`                | Static clusterIP or None for headless services                                                                               | `nil`                                                   |
+
+### Exposing parameters
+
+| Parameter                        | Description                                | Default              |
+| -------------------------------- | ------------------------------------------ | -------------------- |
+| `ingress.enabled`                | Enable ingress controller resource         | `false`              |
+| `ingress.certManager`            | Add annotations for cert-manager           | `false`              |
+| `ingress.annotations`            | Ingress annotations                        | `[]`                 |
+| `ingress.hosts[0].name`          | Hostname for InfluxDB<sup>TM</sup> service | `influxdb.local`     |
+| `ingress.hosts[0].path`          | Path within the url structure              | `/`                  |
+| `ingress.tls[0].hosts[0]`        | TLS hosts                                  | `influxdb.local`     |
+| `ingress.tls[0].secretName`      | TLS Secret (certificates)                  | `influxdb.local-tls` |
+| `ingress.secrets[0].name`        | TLS Secret Name                            | `nil`                |
+| `ingress.secrets[0].certificate` | TLS Secret Certificate                     | `nil`                |
+| `ingress.secrets[0].key`         | TLS Secret Key                             | `nil`                |
```

Comment:
It's missing the new parameters included by @javsalgar recently on every ingress definition:
```
| `ingress.apiVersion`               | Force Ingress API version (automatically detected if not set)                     | ``                             |
| `ingress.path`                     | Ingress path                                                                      | `/`                            |
| `ingress.pathType`                 | Ingress path type                                                                 | `ImplementationSpecific`       |
```

Reply:
done!